pkg/apis/smith/v1/types.go

```diff
@@ -258,3 +291,36 @@ func (rs *ResourceSpec) IntoTyped(obj runtime.Object) error {
 	}
 	return fmt.Errorf("cannot convert %T into typed object %T", rs.Object, obj)
 }
+
+// +k8s:deepcopy-gen=true
+type ResourceStatus struct {
+	Name       ResourceName        `json:"name"`
+	Conditions []ResourceCondition `json:"conditions,omitempty"`
```

Comment:
But wait! This is Conditions[], not Condition[]!

Reply:
(I would prefer all collections are plural - see above)

Reply:
Hm, you are right. We can change it to be consistent.